src/HealthChecks.InfluxDB/DependencyInjection/InfluxDBHealthCheckBuilderExtensions.cs

```diff
@@ -0,0 +1,198 @@
+using HealthChecks.InfluxDB;
+using InfluxDB.Client;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+
+namespace Microsoft.Extensions.DependencyInjection
+{
+    public static class InfluxDBHealthCheckBuilderExtensions
+    {
+        private const string NAME = "InfluxDB";

```

Comment:
```suggestion
        private const string NAME = "influxdb";
```

All HCs use lowercase convention.